keyboards/protokeeb/rev1/keyboard.json

```
@@ -0,0 +1,88 @@
+{
+    "audio": {
+        "default": {
+            "on": false
+        },
+        "pins": ["GP22"]
+    },
+    "board": "GENERIC_RP_RP2040",
+    "bootloader": "rp2040",
+    "debounce": 20,
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [{ "pin_a": "GP12", "pin_b": "GP13", "resolution": 2 }]
+    },
+    "features": {
+        "bootmagic": true,
+        "encoder": true,
+        "extrakey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["GP6", "GP7", "GP8", "GP9", "GP14"],
+        "rows": ["GP2", "GP3", "GP4", "GP5"]
+    },
+    "processor": "RP2040",
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "brightness_steps": 15,
+        "default": {
+            "animation": "static_light",
+            "hue": 0,
+            "on": true,
+            "sat": 255,
+            "speed": 0,
+            "val": 50
+        },
+        "driver": "ws2812",
+        "hue_steps": 15,
+        "led_count": 4,
+        "led_map": [3, 2, 1, 0],
+        "max_brightness": 127,
+        "saturation_steps": 15,
+        "sleep": true
+    },
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x1717"
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP10"
+    },
+    "layouts": {
+        "LAYOUT_custom_matrix": {
```

And update the keymaps accordingly as well. 
```suggestion
        "LAYOUT": {
```

---

I have updated all the keymap files according to the layout's name in the keyboard.json file.